app/controllers/admin/enterprise_fees_controller.rb

```diff
@@ -106,7 +106,7 @@ def check_enterprise_fee_input
           :preferred_flat_percent, :preferred_amount,
           :preferred_first_item, :preferred_additional_item,
           :preferred_minimal_amount, :preferred_normal_amount,
-          :preferred_discount_amount, :preferred_per_unit
+          :preferred_discount_amount, :preferred_per_unit, :preferred_max_items, :preffered_currency
```

Comment:
`preffered_currency `

I think there is a typo on `preffered`

Reply:
Thanks for the review and the comments.  I'll work on it 👍